pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
+    # "meson-python>=0.15.0",
```

Comment:
A small comment could be useful here.
It might be interested to say that we can comment `setuptools` and `wheel` in the future?

Reply:
I removed this actually, this is not needed. We'll add it back when we switch to Meson as our default build tool.